tests/acceptance/test_update_control.py

```diff
@@ -615,3 +616,96 @@ def wait_for_state(state_to_wait_for):
             clear_update_control_maps(connection)
             connection.run("systemctl stop mender-client")
             connection.run("rm -f /data/logger-update-module.log")
+
+    @pytest.mark.min_mender_version("2.7.0")
```

Comment:
It does not matter much, but these could be `3.0.0` to avoid (human) confusion.

Reply:
I agree, but there are many of these, including in mender-image-tests. I'd rather not have them diverge, but fix them all simultaneously.

Reply:
Good point.